Utilities/git-version-gen.cmd

```diff
@@ -91,7 +91,7 @@ for /F %%I IN ('call %GIT% rev-list HEAD --count') do set COMMIT_COUNT=%%I
 
 rem // Check if the current build system sets the git branch and version.
 rem // The name is misleading. This is also used for master builds.
-if defined SYSTEM_PULLREQUEST_SOURCEBRANCH (
```

Comment:
remove the line with the comment above

Reply:
Also remove the same comment in the else branch and adjust the echo with BUILD_SOURCEBRANCHNAME